### DISCUSSION TITLE
Cert out of date (X509) error in Arduino IDE Board Manager

### DISCUSSION THREAD

**Discussion by AutoKitsune** [SpenceKonde/ATTinyCore#846] [General]
Hello.  When I add the URL: ```https://drazzy.com/package_drazzy.com_index.json```  to the ArdIDE 2.3.2 boards manager, I get the following in response:

> Some indexes could not be updated. Get "https://drazzy.com/package_drazzy.com_index.json": tls: failed to verify certificate: x509: certificate has expired or is not yet valid: current time 2024-02-21T07:35:17Z is after 2024-02-16T07:25:21Z: https://drazzy.com/package_drazzy.com_index.json

Not quite sure which end error is...  can anyone assist? 

DId I use the right URL (seems to be OK in browser...)  Is there a workaround? (Sorta Arduino IDE _n00b_, me...)

*** edit *** I did try : ```http://drazzy.com/package_drazzy.com_index.json``` first, and again after, but the message is (almost) the same (note first reference still contains the 'httpsecure' prefix.)

> Some indexes could not be updated. Get "**https://drazzy.com/package_drazzy.com_index.json**": tls: failed to verify certificate: x509: certificate has expired or is not yet valid: current time 2024-02-21T07:XX:XXZ is after 2024-02-16T07:25:21Z: **http://drazzy.com/package_drazzy.com_index.json**

**Comment by hmeijdam** [comment]
try this https://github.com/SpenceKonde/megaTinyCore/issues/1059#issuecomment-1950286469

**Reply by AutoKitsune** [reply]
Aha - a good response for a workaround, thank you.

**Comment by SpenceKonde** [comment]
As soon as I have internet on the computer back I will look at this, all I
have now is the phone which can't run telnet

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***
